docs/BreakingChanges/048 ObsoleteAttribute exports as both ObsoleteAttribute and DeprecatedAttribute in WinMD scenarios.md

```diff
@@ -0,0 +1,24 @@
+## 48: ObsoleteAttribute exports as both ObsoleteAttribute and DeprecatedAttribute in WinMD scenarios
```

Comment:
{
  "Id": "48",
  "Title": "ObsoleteAttribute exports as both ObsoleteAttribute and DeprecatedAttribute in WinMD scenarios",
  "Details": "When you create a Windows Metadata library (.winmd file), the ObsoleteAttribute attribute is exported as both ObsoleteAttribute and Windows.Foundation.DeprecatedAttribute.",
  "Suggestion": "Recompilation of existing source code that uses the ObsoleteAttribute attribute may generate warnings when consuming that code from C++/CX or JavaScript.\n\nWe do not recommend applying both ObsoleteAttribute and Windows.Foundation.DeprecatedAttribute to code in managed assemblies; it may result in build warnings.",
  "Link": "https://msdn.microsoft.com/en-us/library/dn458356(v=vs.110).aspx",
  "Markdown": null,
  "ApplicableApis": [
    "Not detectable via API analysis"
  ],
  "Related": null,
  "VersionBroken": "4.5.1",
  "VersionFixed": null,
  "IsBuildTime": true,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": false,
  "BugLink": null,
  "Notes": null,
  "IsRetargeting": true,
  "ImpactScope": 3
}

**Parsed APIs:**
- [ ] Not detectable via API analysis